products/compute/api.yaml

```diff
@@ -7553,6 +7553,172 @@ objects:
         description: |
           The default port used if the port number is not specified in the
           network endpoint.
+  - !ruby/object:Api::Resource
+    name: 'RegionNetworkEndpointGroup'
+    kind: 'compute#networkEndpointGroup'
+    base_url: 'projects/{{project}}//regions/{{region}}//networkEndpointGroups'
+    input: true
+    min_version: beta
+    has_self_link: true
+    collection_url_key: 'items'
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation': 'https://cloud.google.com/load-balancing/docs/negs/serverless-neg-concepts'
+      api: 'https://cloud.google.com/compute/docs/reference/rest/beta/regionNetworkEndpointGroups'
+    description: |
+      A regional NEG that can support Serverless Products.
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        kind: 'compute#operation'
+        path: 'name'
+        base_url: 'projects/{{project}}/regions/{{region}}/operations/{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'targetLink'
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'status'
+        complete: 'DONE'
+        allowed:
+          - 'PENDING'
+          - 'RUNNING'
+          - 'DONE'
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error/errors'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::ResourceRef
+        name: 'region'
+        resource: 'Region'
+        imports: 'name'
+        description: |
+          A reference to the region where the Serverless NEGs Reside.
+        required: true
+    properties:
+      - !ruby/object:Api::Type::Integer
+        name: 'id'
```

Comment:
since this resource is terraform-only for now, we can just remove this here instead of having it here but excluding it from terraform.yaml

Reply:
I don't follow and the id is an API output field.

Reply:
unless we conventionally ignore the id number from google and use the selflink as id

Reply:
yeah, we typically ignore it completely unless other resources would need to reference it (which they almost always don't). I was just saying that I had noticed that you included the field in api.yaml, but then excluded it in terraform.yaml. It's easier to just not have the field at all in that case.

Reply:
I got it now, looked at the generated markdown and it doesn't appear if i remove it